config/util/identifiers/suffix.json

```diff
@@ -19,8 +19,20 @@
       "@type": "ExtensionBasedMapper",
       "base": { "@id": "urn:solid-server:default:variable:baseUrl" },
       "rootFilepath": { "@id": "urn:solid-server:default:variable:rootFilePath" },
-      "overrideTypes_acl": "text/turtle",
-      "overrideTypes_meta": "text/turtle"
+      "overrideTypes": [
+        {
+          "ExtensionBasedMapper:_overrideTypes_key": "acl",
+          "ExtensionBasedMapper:_overrideTypes_value": "text/turtle"
+        },
+        {
+          "ExtensionBasedMapper:_overrideTypes_key": "meta",
+          "ExtensionBasedMapper:_overrideTypes_value": "text/turtle"
+        },
+        {
+          "ExtensionBasedMapper:_overrideTypes_key": "trig",
+          "ExtensionBasedMapper:_overrideTypes_value": "application/trig"
+        }
+      ]
```

Comment:
Here's me wishing that the object could be reused across `subdomain.json` and `suffix.json`, but I understand why that is currently not possible. Future idea for Components.js to have generic key/value props?